### ISSUE TITLE
DR | Validate old SavedClaim records for evidence errors and reenable DeleteSavedClaimRecordsJob

### ISSUE THREAD

**Issue by dfong-adh** [department-of-veterans-affairs/va.gov-team#93400]
## Value Statement

**_As a_** Veteran

**_I want to_** my submission data to be removed when no longer needed

**_So that_** my privacy is protected

---

## Background Context

- #84348

## Outcome, Success Measure, KPI(S), and Tracking Link

- Any data that is no longer needed is removed from the database

## Acceptance Criteria

- [x] Any submission data is removed once the 60 day window is complete

## Designs and Build Notes

- Add here

### Enablement team (if needed)
@ member of team

## Out of scope

- Add here

## Open questions

- Add here

## Tasks

- [ ] _Task_
- [ ] _Task_
- [ ] _Task_

## Definition of Ready

- [ ] Clear value description
- [ ] Testable acceptance criteria
- [ ] Accessibility added to acceptance criteria
- [ ] Approved designs attached
- [ ] Sample data provided where appropriate
- [ ] Estimated to fit within the sprint
- [ ] Dependencies and blockers linked

## Definition of Done

- [ ] Meets acceptance criteria
- [ ] Passed E2E testing (90% coverage)
- [ ] Passed unit testing (90% coverage)
- [ ] Passed integration testing (if applicable)
- [ ] Code reviewed (internal)
- [ ] Submitted to staging
- [ ] Team approved production verification process
- [ ] Reviewed and approved by product and/or design

**Comment by Mottie** [comment]
Going through old records first, then re-enabling save-claims-deletion job

**Comment by Mottie** [comment]
Should have a PR ready today. Then run the rake test and clear out old data.

**Comment by Mottie** [comment]
Derek to check if he has production write access in ArgoCD & run migration.

**Comment by dfong-adh** [comment]
Verified production write access. Will set up a meeting to run the data migration script in prod.

**Comment by dfong-adh** [comment]
Data migration executed in production. Values verified.

**Comment by dfong-adh** [comment]
Flipper for delete job re-enabled.
